### PR TITLE
Feature/csh 5617 rewrite readme

### PR DESCRIPTION
**Pull request by wwjhu**

### CODE REVIEW

**Comment by kgeisink** [comment]
Something I noticed in general in the readme is the use of the term "custom components". I always thought that, while it was the name of the feature, once it is released everything will be a custom component so we might as well just call it components, or in this case interactive components if applicable. I am not completely sure whether I got all of them in the README though.

**Comment by wwjhu** [comment]
The problem with interactive components is they consist of two parts: the external part (what's loaded in the iframe) and the glue part that integrates it into the editor. I tried to consistently name the external part the interactive component, while the glue part is named custom component. Suggestions for better naming are very welcome.

File: README.md [hunk 1]
```diff
@@ -150,4 +151,26 @@ The sdk.js contains a function to automatically adjust the component height to t
 }
 ```
 
+## Incorporating changes from the boilerplate
```
Comment (wwsno) [inline README.md]: These instructions are for ourself?

**Comment by wwsno** [comment]
I agree with Koen to just call them "components" and leave out the "custom" part.

File: README.md (hunk 1 above)
Comment (wwjhu) [reply README.md]: Although externals could do the same (and send a PR), I expect that it will be us to do this mostly. Better ideas where to put it?

**Pull request merged**